### PR TITLE
Bug fixes for running locally

### PR DESCRIPTION
**Pull request by rjtayl**
A few small changes and the return of a missing function to allow develop to run bug free again.

### CODE REVIEW

**Comment by buzinsky** [comment]
Are you actually getting energy spectra? Can you show a histogram of what you are getting?
I am getting a bunch of NaNs

You can get rid of the commented lines - the git history will still have it if we need to go back (seems unlikely)


Screenshot 2023-08-28 at 10 50 59 AM

[test.py.txt](https://github.com/Helium6CRES/he6-cres-spec-sims/files/12457013/test.py.txt)

**Comment by rjtayl** [comment]
I am not seeing that error. I tried running a much higher statistics run and am now seeing rarely a integration nonconvergence warning:
`IntegrationWarning: The algorithm does not converge.  Roundoff error is detected
  in the extrapolation table.  It is assumed that the requested tolerance
  cannot be achieved, and that the returned result (if full_output = 1) is
  the best which can be obtained.
  T_a = 4 / velocity(energy) * integrate.quad(sec_theta, 0, zmax)[0]`

However, this warning does not appear to break anything like yours. 

Here is a histogram of energies. Looks like we may be calling cobs incorrectly. 
image

**Comment by buzinsky** [comment]
@rjtayl Did you have other changes in the works? Or should I test it so that we can finalize/ merge this commit
Can you explain why the beta spectra was broken before and now working?

**Comment by rjtayl** [comment]
@buzinsky You can go ahead and test. The spectra are being pulled correctly, the shape we see is different because we always test the same number of betas at each field.

File: .gitignore
Comment (buzinsky) [inline .gitignore:1]: I don't understand what happened here. Please be careful when committing (eg. not doing git commit -A unless sure.)
You could include, for example, 
*.pyc
cres_specs_env

etc.
But we don't need 8,000 lines for this.

File: test.png
Comment (buzinsky) [inline test.png:1]: Please remove this file from the repo. Added to the PR thread.

**Comment by buzinsky** [comment]
Done with my comments, hopefully you got them. Once those two (.png & gitignore) are resolved, you are good to merge

test

File: .gitignore
Comment (rjtayl) [reply .gitignore:1]: These were supposed to be in my git ignore... its just my testing environment.  Sorry about that will remove.

**Review by buzinsky — APPROVED** [review]

**Pull request merged**